contracts/hydro/src/migration/v2_0_0.rs

```diff
@@ -0,0 +1,271 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Decimal, DepsMut, Env, Order, StdError, Storage, Timestamp, Uint128};
+use cw_storage_plus::{Item, Map};
+use neutron_sdk::bindings::query::NeutronQuery;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::contract::{compute_current_round_id, compute_round_end, get_lock_time_weighted_shares};
+use crate::error::ContractError;
+
+use crate::lsm_integration::validate_denom;
+use crate::migration::v1_1_0::{ConstantsV1_1_0, ProposalV1_1_0, VoteV1_1_0};
+use crate::state::{CONSTANTS, LOCKS_MAP};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct MigrateMsgV2_0_0 {
+    pub max_bid_duration: u64,
+}
+
+#[cw_serde]
+pub struct ConstantsV2_0_0 {
```

Comment:
These data structures are currently the same as the ones in state.rs code on main. As we keep adding new features and changing those data structures, we will need to have the structures the way they are in the state of V2 deployed contracts, once we cut the V2 release. At that point, we should create a snapshot of V2 data structures to be able to write future migrations. I did this in advance. Let me know if you think we should save this step for later and use Constants, etc. from state.rs here instead.